compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -292,6 +292,11 @@ uint32_t cal_offset(loco::TensorShape &dimension, uint32_t *indices)
          indices[2] * dimension.dim(3).value() + indices[3];
 }
 
+uint32_t cal_offset_2d(loco::TensorShape &dimension, uint32_t *indices)
+{
+  return indices[0] * dimension.dim(1).value() + indices[1];
```

Comment:
I don't know caller is calling with safe index.
It would be better to add some assertion check before accessing ptr and array.